sql/catalyst/src/main/scala/org/apache/spark/sql/internal/StaticSQLConf.scala

```diff
@@ -140,7 +140,8 @@ object StaticSQLConf {
   val QUERY_EXECUTION_LISTENERS = buildStaticConf("spark.sql.queryExecutionListeners")
     .doc("List of class names implementing QueryExecutionListener that will be automatically " +
       "added to newly created sessions. The classes should have either a no-arg constructor, " +
-      "or a constructor that expects a SparkConf argument.")
+      "or a constructor that expects a SparkConf argument. User can use SQLConf.get() to get the " +
+      "corresponding SessionState's conf when initialize QueryExecutionListener.")
```

Comment:
I wouldn't publicly document it for now though since `SQLConf.get` isn't an API technically.

Reply:
Hmm, ok. But it's really a useful tips when use query listener.

Reply:
Yeah but let's don't document the internal stuff. I think it's a pretty corner case in any event.

Reply:
> Yeah but let's don't document the internal stuff. I think it's a pretty corner case in any event.

All right, let me close this.